src/data.rs

```diff
@@ -206,13 +205,19 @@ impl Data {
                 .call_indirect(passive_fn_alloc_type, passive_tid);
 
             // update the existing passive_alloc function export with the new function body
-            let passive_alloc_fid = get_exported_func(module, "passive_alloc")?;
+            let passive_alloc_fid = module.exports.get_func_by_name("passive_alloc")?;
             let passive_alloc_func = module.funcs.get_mut(passive_alloc_fid);
             passive_alloc_func.kind =
                 FunctionKind::Local(builder.local_func(vec![passive_idx, offset_local, len_local]));
         }
 
-        remove_exported_func(module, "passive_alloc")?;
+        module.exports.delete(
```

Comment:
This isn't *great*, I kind of would like a `delete_by_name` here that does the same thing, inside `ModuleExports`...

Reply:
Yeah a delete_by_name seems like a good idea

Reply:
Will add this!

- [ ] Add `delete_by_name` to `ModuleExports`/`ModuleImports`

Reply:
Added in to the [ongoing `walrus` branch](https://github.com/rustwasm/walrus/pull/250/commits/b418dea3081395627d9b633cd0718983ea6db21f)